### PR TITLE
feat: Add individual bundler packages

### PR DESCRIPTION
**Pull request by lforst**

### CODE REVIEW

File: packages/eslint-configs/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
-  "name": "eslint-config-local",
+  "name": "@sentry-internal/eslint-config",
```
Comment (Lms24) [inline packages/eslint-configs/package.json:2]: Does this still work? IIRC you said yesterday that the package needs to be named `eslint-config` (?)
Or does it work b/c scope doesn't count?

File: packages/esbuild-plugin/.babelrc.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/env", "@babel/typescript"]
+}
```
Comment (Lms24) [inline packages/esbuild-plugin/.babelrc.json:3]: Totally optional: 
I think the values here are the same for all projects, right? Can we extract this file into one package?

File: packages/esbuild-plugin/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.(t|j)sx?$": ["@swc/jest"],
+  },
+};
```
Comment (Lms24) [inline packages/esbuild-plugin/jest.config.js:6]: Totally optional:
Any change, we can also extract this to a central package?
We might wanna add a "dev-utils" package (I know this sounds oddly familiar....)

File: packages/esbuild-plugin/rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import resolve from "@rollup/plugin-node-resolve";
```
Comment (Lms24) [inline packages/esbuild-plugin/rollup.config.js:1]: Totally optional:
Might also be a candidate for a centralized config?

File: packages/rollup-plugin/.gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+dist
```
Comment (Lms24) [inline packages/rollup-plugin/.gitignore:1]: Optional: 
Hmmm WDYT about adding something like `packages/*/dist` into the root .gitignore instead?

**Review by Lms24 — APPROVED** [review]
🚀 
(Just some minor optional nits but totally happy to revisit this later)

File: packages/rollup-plugin/.gitignore (hunk 5 above)
Comment (lforst) [reply packages/rollup-plugin/.gitignore:1]: More risky/opaque as to why something is "gitignored". I'll leave it for now.

File: packages/esbuild-plugin/jest.config.js (hunk 3 above)
Comment (lforst) [reply packages/esbuild-plugin/jest.config.js:6]: If we notice that having stuff duplicated becomes a problem we can extract this. I usually don't like drying things too early. Same goes for other comments.

**Pull request merged**